orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton.go

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reindex
```

Comment:
https://magma.github.io/magma/docs/next/contributing/contribute_conventions#:~:text=Default%20to%20using%20separate%20_test%20package%20for%20tests

Reply:
? I think this isn't the test package? I do name the package of the test `reindex_test`? Don't quite understand what we want to change